<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeStatusWidget</class>
 <widget class="QWidget" name="TreeStatusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: none;
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetStatus" native="true">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgba(0,0,0,50);
border-radius: 8px;
</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <bold>false</bold>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>8</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="lblBrand">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Brand</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblModel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Model</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>7</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Fahrzeugnummer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCarNum">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>00000000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblPic">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
